fe/fe-core/src/main/java/com/starrocks/external/iceberg/cost/IcebergTableStatisticCalculator.java

```diff
@@ -96,16 +93,13 @@ private Statistics makeTableStatistics(List<Expression> icebergPredicates,
         LOG.debug("Begin to make iceberg table statistics!");
         List<Types.NestedField> columns = icebergTable.schema().columns();
         IcebergFileStats icebergFileStats = generateIcebergFileStats(icebergPredicates, columns);
-        if (icebergFileStats == null) {
-            return Statistics.builder().build();
```

Comment:
why remove this? i wonder if this will cause wrong plan or not

Reply:
`makeTableStatistics` or `makeColumnStatistics`  is `return null` will cause  compute statistics fail for empty table.

but in my cause,  only call  `makeColumnStatistics` function . so `makeTableStatistics` can return null statistics ?